src/cfn/approval/cli.ts

```diff
@@ -22,6 +22,11 @@ export function buildApprovalCommands(args: Argv, commands = lazy): Argv {
     description('request template approval'),
     (args) => args
       .demandCommand(0, 0)
+      .option('lint-template', lintTemplateOpt)
+      .option('use-parameters', {
```

Comment:
How about `s/use-parameters/lint-using-parameters/` as the argument only make sense there?